dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/config_button_group.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { uniqueId } from 'lodash';
+import { EuiTitle, EuiSpacer, EuiButtonGroup } from '@elastic/eui';
+interface ToggleButtonOptions {
+  id: string;
+  label: string;
+}
+interface ToggleGroupProps {
+  title: string;
+  legend: string;
+  groupOptions: ToggleButtonOptions[];
+  idSelected: string;
+  handleButtonChange: (id: string, value?: any) => void;
+}
+export const ButtonGroupItem: React.FC<ToggleGroupProps> = ({
+  title, legend, groupOptions, idSelected, handleButtonChange
+}) => (
+  <>
+    <EuiTitle size="xxs">
+      <h3>{title}</h3>
+    </EuiTitle>
+    <EuiSpacer size="s" />
+    <div style={{ width: "fit-content" }}>
+      <EuiButtonGroup
+        id={uniqueId('button-select-')}
```

Comment:
Please use html id generator moving forward for ids, and gradually replace all other uniqueId with that. [Guide](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#how-to-generate-ids), [html id generator](https://eui.elastic.co/#/utilities/html-id-generator)